app/Resources/views/newsletter/invitation.html.twig

```diff
@@ -29,7 +29,7 @@
                 </p>
 
                 <div class="newsletter-page__form l__wrapper-500">
-                    {{ form_start(invitation_form) }}
+                    {{ form_start(invitation_form, {attr: {novalidate: 'novalidate'}}) }}
```

Comment:
I think there are indeed.

First, validation is not consistent across browsers and is done server side; second, from a FormType perspective the tree is resized dynamically depending on the submitted data and according to this point of view every field is required and must be validated, changing the configuration of children would expose potential side effects on validation and empty data handling, I would not advise it.